extensions/wikia/SpecialDiscussionsLog/SpecialDiscussionsLogController.class.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+/**
+ * Discussion user log page
+ */
+class SpecialDiscussionsLogController extends WikiaSpecialPageController {
+	// @todo
```

Comment:
What about thins TODO? 
I believe we should resolve it or create a ticket to address that.